### PR TITLE
adds listserv signup to About page

### PR DESCRIPTION
**Pull request by tgfisher**

### CODE REVIEW

**Comment by eshedmargalit** [comment]
@tgfisher I think we should have some sort of confirmation that the email was successfully submitted, otherwise folks might try to enter it multiple times or think it didn't go through

File: src/components/ListservSignup/ListservSignup.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.seatbutton {
```
Comment (eshedmargalit) [inline src/components/ListservSignup/ListservSignup.css]: Hmm I don't love this solution, it doesn't scale well with screen size
I'd consider re-writing the formatting with some combination of reacstraps <Row> and <Col> components

Comment (tgfisher) [reply src/components/ListservSignup/ListservSignup.css]: Yea I didn't like it either. I'll take another look at it with your suggestion in mind. We should chat after the studio Sunday.

**Comment by tgfisher** [comment]
1. Improved form layout
2. Button gives feedback while submitting and after a valid submission.

**Comment by tgfisher** [comment]
1. Improved form layout
2. Button gives feedback while submitting and after a valid submission.

**Pull request merged**